<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1" language="da" sourcelanguage="en">
<context>
    <name>Cartogram</name>
    <message>
        <location filename="../cartogram3.py" line="112"/>
        <source>&amp;cartogram3</source>
        <translation type="obsolete">&amp;Cartogram</translation>
    </message>
    <message>
        <location filename="../cartogram3.py" line="209"/>
        <source>Compute cartogram</source>
        <translation type="obsolete">Opret cartogram</translation>
    </message>
    <message>
        <location filename="../cartogram3.py" line="111"/>
        <source>&amp;Cartogram</source>
        <translation type="obsolete">&amp;Cartogram</translation>
    </message>
    <message>
        <location filename="../cartogram3.py" line="485"/>
        <source>Error</source>
        <translation type="obsolete">Fejl</translation>
    </message>
    <message>
        <location filename="../cartogram3.py" line="279"/>
        <source>You need at least one polygon vector layer to create a cartogram.</source>
        <translation type="obsolete">Du skal bruge mindst ét vektorlag for at kunne oprette et cartogram.</translation>
    </message>
    <message>
        <location filename="../cartogram3.py" line="357"/>
        <source>Cancel</source>
        <translation type="obsolete">Annuller</translation>
    </message>
    <message>
        <location filename="../cartogram3.py" line="466"/>
        <source>cartogram3 successfully finished computing a cartogram for field ‘{fieldName}’ after {iterations} iterations with {avgError:.2n}% average error remaining.</source>
        <translation type="obsolete">cartogram3 succesfuldt afsluttet beregne en cartogram for felt ‚{fieldName}‘ efter {iterations} iterationer med en gennemsnitlig fejl af {avgError:.2n}% tilbage.</translation>
    </message>
    <message>
        <location filename="../cartogram3.py" line="477"/>
        <source>cartogram3 computation cancelled by user</source>
        <translation type="obsolete">cartogram3: oprettelse af cartogram stoppet af brugeren</translation>
    </message>
    <message>
        <location filename="../cartogram3.py" line="485"/>
        <source>An error occurred during cartogram creation. Please see the ‘Plugins’ section of the message log for details.</source>
        <translation type="obsolete">Der opstod en fejl. Se venligst &amp;quot;Plugin&amp;quot;-loggen for flere detaljer.</translation>
    </message>
    <message>
        <location filename="../cartogram3.py" line="286"/>
        <source>Add sample dataset</source>
        <translation type="obsolete">Tilføj eksempeldata</translation>
    </message>
</context>
<context>
    <name>CartogramDialog</name>
    <message>
        <location filename="../ui/cartogram_dialog.ui" line="14"/>
        <source>cartogram3</source>
        <translation>cartogram3</translation>
    </message>
    <message>
        <location filename="../ui/cartogram_dialog.ui" line="20"/>
        <source>Input layer:</source>
        <translation>Lag:</translation>
    </message>
    <message>
        <location filename="../ui/cartogram_dialog.ui" line="27"/>
        <source>Field(s):</source>
        <translation>Felt(er):</translation>
    </message>
    <message>
        <location filename="../ui/cartogram_dialog.ui" line="41"/>
        <source>Stop conditions:</source>
        <translation>Betingelser:</translation>
    </message>
    <message>
        <location filename="../ui/cartogram_dialog.ui" line="57"/>
        <source>max. number of iterations:</source>
        <translation>max. iterationer:</translation>
    </message>
    <message>
        <location filename="../ui/cartogram_dialog.ui" line="77"/>
        <source>max. average error:</source>
        <translation>max. gennemsnitlig fejl:</translation>
    </message>
    <message>
        <location filename="../ui/cartogram_dialog.ui" line="84"/>
        <source>%</source>
        <translation>%</translation>
    </message>
    <message>
        <location filename="../ui/cartogram_dialog.ui" line="97"/>
        <source>Calculation stops as soon as one condition is met.</source>
        <translation>Beregningen slutter, når en betingelse er opfyldt.</translation>
    </message>
</context>
<context>
    <name>CartogramProcessingAlgorithm</name>
    <message>
        <location filename="../lib/cartogramprocessingalgorithm.py" line="59"/>
        <source>Compute cartogram</source>
        <translation type="unfinished">Opret cartogram</translation>
    </message>
    <message>
        <location filename="../lib/cartogramprocessingalgorithm.py" line="53"/>
        <source>Vector geometry</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../lib/cartogramprocessingalgorithm.py" line="62"/>
        <source>Input layer</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../lib/cartogramprocessingalgorithm.py" line="103"/>
        <source>Field</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../lib/cartogramprocessingalgorithm.py" line="77"/>
        <source>max. number of iterations</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../lib/cartogramprocessingalgorithm.py" line="85"/>
        <source>max. average error (%)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../lib/cartogramprocessingalgorithm.py" line="96"/>
        <source>Output layer</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../lib/cartogramprocessingalgorithm.py" line="109"/>
        <source>Iterations needed to meet residual error threshold.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../lib/cartogramprocessingalgorithm.py" line="115"/>
        <source>Residual average error.</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>CartogramProcessingProvider</name>
    <message>
        <location filename="../lib/cartogramprocessingprovider.py" line="25"/>
        <source>Cartogram</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>CartogramUserInterfaceMixIn</name>
    <message>
        <location filename="../lib/cartogramuserinterfacemixin.py" line="94"/>
        <source>Cartogram of {:s}, distorted using ‘{:s}’</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../lib/cartogramuserinterfacemixin.py" line="128"/>
        <source>Cancelled</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../lib/cartogramuserinterfacemixin.py" line="135"/>
        <source>&amp;Cartogram</source>
        <translation type="unfinished">&amp;Cartogram</translation>
    </message>
    <message>
        <location filename="../lib/cartogramuserinterfacemixin.py" line="140"/>
        <source>Compute cartogram</source>
        <translation type="unfinished">Opret cartogram</translation>
    </message>
    <message>
        <location filename="../lib/cartogramuserinterfacemixin.py" line="177"/>
        <source>Add sample dataset</source>
        <translation type="unfinished">Tilføj eksempeldata</translation>
    </message>
    <message>
        <location filename="../lib/cartogramuserinterfacemixin.py" line="169"/>
        <source>Error</source>
        <translation type="unfinished">Fejl</translation>
    </message>
    <message>
        <location filename="../lib/cartogramuserinterfacemixin.py" line="171"/>
        <source>You need at least one polygon vector layer to create a cartogram.</source>
        <translation type="unfinished">Du skal bruge mindst ét vektorlag for at kunne oprette et cartogram.</translation>
    </message>
    <message>
        <location filename="../lib/cartogramuserinterfacemixin.py" line="195"/>
        <source>Computing cartogram</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../lib/cartogramuserinterfacemixin.py" line="204"/>
        <source>Cancel</source>
        <translation type="unfinished">Annuller</translation>
    </message>
</context>
<context>
    <name>CartogramWorker</name>
    <message>
        <location filename="../cartogram_worker.py" line="103"/>
        <source>Iteration {i}/{mI} for field â{fN}â</source>
        <translation type="obsolete">Iteration {i}/{mI} for felt ‚{fN}‘</translation>
    </message>
</context>
</TS>
